framework/assets/yii.js

```diff
@@ -272,12 +272,13 @@ yii = (function ($) {
         },
 
         getQueryParams: function (url) {
+            url = url || document.location.href;
```

Comment:
Are we allow calling `getQueryParams` without arguments?
This change will just hide possible developer mistake.
Does not seem to be right.